web-client/src/presenter/computeds/formattedTrialSessions.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```

Comment:
Nit: Time to split this into two files?

Reply:
Good idea! It is now split. filterFormattedSessionsByStatus was already pulled out before, and now formatSession is as well. the es lint disable max lines is no longer needed in formattedTrialSessions.test.js~